### ISSUE TITLE
Main branch install error due to too long version 

### ISSUE THREAD

**Issue by Kaik** [zikula/core#4831]
### Expected behaviour - What did you expect to happen?

```bash
Successful installation
```

### Current behavior - What happened instead?

```bash
Activating system extensions
14:20:55 CRITICAL  [console] Error thrown while running command "zikula:install:finish". Message: "An exception occurred while executing 'INSERT INTO extension_deps (modid, modname, minversion, maxversion, status) VALUES (?, ?, ?, ?, ?)' with params [17, "php-translation\/symfony-bundle", "dev-extractor-service-definitions", "dev-extractor-service-definitions", 1]:\n\nSQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'minversion' at row 1" ["exception" => Doctrine\DBAL\Exception\DriverException^ { …},"command" => "zikula:install:finish","message" => """  An exception occurred while executing 'INSERT INTO extension_deps (modid, modname, minversion, maxversion, status) VALUES (?, ?, ?, ?, ?)' with params [17, "php-translation\/symfony-bundle", "dev-extractor-service-definitions", "dev-extractor-service-definitions", 1]:\n  \n  SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'minversion' at row 1  """]
```
```


### Zikula version

main

### PHP version

7.4.x

**Comment by Guite** [comment]
Can you check if this also affects the `3.1` branch?
Also: does it only affect direct core installation or also distribution based installations (e.g. the `3.1.0` release)?

**Comment by Kaik** [comment]
Looks like 3.1 is affected https://github.com/zikula/core/blob/3.1/composer.json#L61
I'm not sure about distribution based installations I don't have env for those. I have checked `3.1.0` files and that entry does not exist in composer.json

**Comment by Guite** [comment]
fixed and re-released 3.1.0
